lib/helper/REST.js

```diff
@@ -1,7 +1,8 @@
+const Helper = require('../helper');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [`../helper` import should occur after import of `chai`](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4763679)